bin/codemods/src/assign-component-to-displayname-variable.js

```diff
@@ -17,30 +17,19 @@
  and
 
  export default someHoC( class extends {Component|PureComponent|React.Component} {
-	static displayName = 'SomeDisplayName';
+    static displayName = 'SomeDisplayName';
  } );
 
  to
 
  class SomeDisplayName extends {Component|PureComponent|React.Component} {
-	static displayName = 'SomeDisplayName';
+    static displayName = 'SomeDisplayName';
  };
 
  export default someHoC( SomeDisplayName );
  */
 
-const createClassIdentifier = {
-	type: 'CallExpression',
-	callee: {
-		type: 'MemberExpression',
-		object: {
-			name: 'React',
-		},
-		property: {
-			name: 'createClass',
-		},
-	},
-};
+import * as _ from 'lodash';
```

Comment:
I'd thought about using lodash but noticed that jscodeshift covers the methods needed.
I do prefer it though, myself and every other dev is already comfortable with and it gets rid of the need to then access `.value` after `get`ing :)
Do you have any issue with destructuring the methods to avoid the `_.` noise?

Reply:
personally i prefer `_` over destructuring for a few reasons:
1. namespacing is a good thing: there are some lodash methods that share a name with ones that are also WebAPIs.  Have you ever forgotten to import `find` yet your code works anyway but doesn't behave as expected? Thats because of [Window.find](https://developer.mozilla.org/en-US/docs/Web/API/Window/find).
2. changing the destructuring line as you change which methods you use from lodash is kind of annoying
3. IDE support.  By calling it directly from `_` I get autocomplete in my IDE.


The "benefits" when using webpack _could_ be tree shaking assuming we aren't using the whole lib don't exist here.